README.md

```diff
@@ -1 +1,89 @@
-# lodge
\ No newline at end of file
+# lodge
+
+A simple log lib used by Neoway's ML Platform team.
+
+## Features
+
+* Easy to use
+* Env var configurations
+* Structured production log ready!
+* Options to customize fields
+* Set log level by module
+* Based on stdlib
+
+## Getting Started
+
+```python
+from lodge import log
```

Comment:
@mfrata python's stdlib uses `logging`, loguru uses `logger`, and you're proposing the use of `log`. Maybe it's better to use a previous pattern for making easier to getting started. Loguru's approach seems a better choice.

Reply:
I prefer log because it's shorter, as It's made in some another languages, like Golang https://golang.org/pkg/log/

Usually I do `from loguru import logger as log` too. 

But this it seems a matter of taste. The unique downside I see on using `log` it's not the previous pattern, but it's maybe the confusion with math.log and possible name conflict on global import, like:

```
from lodge import log
from math import log
```

Reply:
@ryukinix i understand your point of view, but my point is not only related to the taste. Let me explain.

One of the aspects during naming is how the people from that community are used to it. If this package is inspired by loguru (easy to use, one import configures everything, etc.) maybe could be interesting use the same name convention.

Reply:
I see your point now, @brenocosta0901. In that case, I agree with you: we should use `logger`.

Reply:
I understand @brenocosta0901 but partly disagree. We created the package borrowing some ideas from previous ones to solve a specific problem for us. Better if solves someone else's problem 😃. We don't have to follow all [conventions](https://github.com/NeowayLabs/lodge/pull/1#discussion_r599190049) from the community. But i said "partly" because you are right in that [logger](https://github.com/search?q=logger.info+language%3APython&type=Code&ref=advsearch&l=Python&l=) is more used than  [log](https://github.com/search?l=Python&q=log.info&type=Code).

I'm changing the doc to logger. But will keep the alias `log = logger`. Since, I share the same opinion as Manoel's.